src/test/java/se/liu/ida/hefquin/engine/wrappers/lpgwrapper/impl/LPG2RDFConfigurationReaderTest.java

```diff
@@ -0,0 +1,144 @@
+package se.liu.ida.hefquin.engine.wrappers.lpgwrapper.impl;
+
+
+import org.apache.jena.graph.Node;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.riot.Lang;
+import org.apache.jena.riot.RDFParser;
+import org.apache.jena.riot.RDFParserBuilder;
+import org.junit.Test;
+import se.liu.ida.hefquin.engine.wrappers.lpgwrapper.data.impl.LPGNode;
+
+import static org.junit.Assert.*;
+
+public class LPG2RDFConfigurationReaderTest {
+
+    @Test
+    public void LPG2RDFConfigWithIRIBasedNodeMapping() {
+        final String turtle =
+                "PREFIX lr:     <http://www.example.org/se/liu/ida/hefquin/lpg2rdf#>\n"
+              + "PREFIX ex:     <http://example.org/>\n"
+              + "\n"
+              + "ex:LPGtoRDFConfig\n"
+              + "   a  lr:LPGtoRDFConfiguration ;\n"
+              + "   lr:labelPredicate  <http://www.w3.org/2000/01/rdf-schema#label> ;\n"
```

Comment:
That's not right. The object of this triple cannot be the URI but it must be a literal that has the URI as its string value and that has the datatype `xsd:anyURI`. Hence, this line should look as follows (but then you also need to introduce the `xsd:` prefix in the header of this snippet of Turtle).
```suggestion
              + "   lr:labelPredicate  "http://www.w3.org/2000/01/rdf-schema#label"^^xsd:anyURI ;\n"
```

Reply:
Same issue then also below in the other tests.